docs/src/background/duality.md

```diff
@@ -149,35 +192,14 @@ A pair of primal-dual variables $(x^\star, y^\star)$ is optimal if
   ```math
   \max_{y_i \in \mathcal{C}_i^*} L(x^\star, y).
   ```
-  That is, for all ``i = 1, \ldots, m``, ``\frac{1}{2}x^TQ_ix + a_i^T x + b_i`` is
-  either zero or in the normal cone of ``\mathcal{C}_i^*`` at ``y^\star``.
-  For instance, if ``\mathcal{C}_i`` is ``\{ x \in \mathbb{R} : x \le 0 \}``, this means that
-  if ``\frac{1}{2}x^TQ_ix + a_i^T x + b_i`` is nonzero at ``x^\star`` then ``y_i^\star = 0``.
-  This is the classical complementary slackness condition.
+  That is, for all ``i = 1, \ldots, m``, ``\frac{1}{2}x^TQ_ix + a_i^T x + b_i``
+  is either zero or in the
+  [normal cone](https://en.wikipedia.org/wiki/Normal_cone) of
+  ``\mathcal{C}_i^*`` at ``y^\star``. For instance, if ``\mathcal{C}_i`` is
+  ``\{ z \in \mathbb{R} : z \le 0 \}``, this means that if
+  ``\frac{1}{2}x^TQ_ix + a_i^T x + b_i`` is nonzero at ``x^\star`` then
+  ``y_i^\star = 0``. This is the classical complementary slackness condition.
 
 If ``\mathcal{C}_i`` is a vector set, the discussion remains valid with
 ``y_i(\frac{1}{2}x^TQ_ix + a_i^T x + b_i)`` replaced with the scalar product
 between ``y_i`` and the vector of scalar-valued quadratic functions.
-
-!!! note
-    For quadratic programs with only affine constraints, the optimality condition
```

Comment:
did you remove the affine constraint section altogether?

Reply:
I moved it up